docker-compose.yml

```diff
@@ -7,5 +7,6 @@ services:
     volumes:
       - "./data:/opensanctions/data"
       - "./opensanctions:/opensanctions/opensanctions"
+      - "./datasets:/opensanctions/datasets"
```

Comment:
I don't think we want to change this generally right now. To avoid having a modified version lying around, I think you can use the --volume argument to `docker compose run`